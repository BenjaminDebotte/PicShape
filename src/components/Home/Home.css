hr { width: 30%;}
h1 {font-size: 100px !important;}

@keyframes transition {
0%   { margin-top: -70px;opacity:0; }
100% { margin-top: 0;opacity:1; }
}

.middle {opacity:0;animation: transition 1s ease-in-out 0.5s forwards;}
