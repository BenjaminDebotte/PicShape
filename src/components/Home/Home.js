import React, { Component } from 'react';
import './Home.css';
import { Link } from 'react-router';

export class Home extends Component {

  constructor(props) {
      super(props);
      this.state = {
    }
  }

  render() {
      return (
                <div className="ui">
                  <h1 className="ui huge header inverted">PicShape</h1>
                  <hr></hr>
                  <Link className="ui button blue" to="/upload"> Go! </Link>
              </div>
      );
  }
}
  export default Home;
