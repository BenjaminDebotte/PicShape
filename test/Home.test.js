import React from 'react';
import  {Home} from '../src/components/Home/Home';
import renderer from 'react-test-renderer';

describe('components', function() {
  describe('<Home />', function() {
    it('renders correctly', function() {
      var tree = renderer.create(<Home />).toJSON();
      expect(tree).toMatchSnapshot();
        });
  });
});
