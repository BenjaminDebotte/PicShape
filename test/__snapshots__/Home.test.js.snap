exports[`components <Home /> renders correctly 1`] = `
<div
  className="ui">
  <h1
    className="ui huge header inverted">
    PicShape
  </h1>
  <hr />
  <a
    className="ui button blue"
    onClick={[Function bound handleClick]}
    style={Object {}}>
     Go! 
  </a>
</div>
`;
